Assets/PatchKit Patcher/Scripts/AppData/Local/UnityCache.cs

```diff
@@ -1,19 +1,38 @@
-﻿using PatchKit.Unity.Utilities;
+﻿using PatchKit.Logging;
+using PatchKit.Unity.Patcher.Debug;
+using PatchKit.Unity.Utilities;
 using UnityEngine;
 
 namespace PatchKit.Unity.Patcher.AppData.Local
 {
-    class UnityCache : ICache
+    public class UnityCache : ICache
     {
+        private readonly string _hashedSecret;
+
+        public UnityCache(string appSecret)
+        {
+            _hashedSecret = HashSecret(appSecret);
+        }
+
+        private string HashSecret(string secret)
+        {
+            return HashCalculator.ComputeStringHash(secret);
```

Comment:
Please don't use xxHash for these kind of hashing, instead check what hashing algorithms are build-in in standard library and use it. I would recommend using MD5.

Reason: The main purpose of using xxHash to hash application's content is the hashing speed. If for some reason we will be forced to change this algorithm, it would be a problem to completely remove it from the patcher if there's an usage of this algorithm besides its main purpose.